frontend/src/metabase-lib/query-utils.ts

```diff
@@ -0,0 +1,39 @@
+import { aggregations } from "./aggregation";
+import { breakouts } from "./breakout";
+import { expressions } from "./expression";
+import { fields } from "./fields";
+import { filters } from "./filter";
+import { joins } from "./join";
+import { hasLimit } from "./limit";
+import { orderBys } from "./order_by";
+import { stageCount } from "./query";
+import type { Query } from "./types";
+
+export const hasAnyClauses = (query: Query): boolean => {
+  return stageCount(query) > 1 || hasAnyClausesAtStage(query, 0);
```

Comment:
This check seems incorrect btw.

Reply:
ace95110fb774ba2adf20c695d4771eb22ff36e7